content/account/organizations.textile

```diff
@@ -0,0 +1,178 @@
+<p>Users provisioned through SCIM cannot modify their name or email address within Ably. All personal information updates must be made through your identity provider, and changes will sync to Ably upon the next SCIM update cycle.</p>
+</aside>
+
+h3(#permissions). User permissions
+
+A user's total permissions consist of:
+
+* Roles and permissions assigned directly to the user.
+* Roles and permissions granted to all groups the user is part of.
+
+h2(#scim). Provision users via SCIM
+
+<aside data-type='note'>
+<p>Google Workspace _alone_ does not natively support SCIM.</p>
+</aside>
+
+"Manage access":/account/users to multiple Ably accounts through a single identity provider. To enable this, configure both "SSO":/account/sso with your chosen identity provider and "SCIM":#SCIM. Once configured, Ably automatically provisions and deprovisions users with access to the Ably app in your identity provider, either individually or through assigned groups. New users are added to Ably's default provisioning account with the role of *Developer*.
+
+<aside data-type='important'>
+	<p>Ably will reject any SCIM-provisioned user whose email address hasn't been pre-approved and added to your organization by Ably support. Ensure all users you intend to provision are registered with Ably beforehand.</p>
+	</aside>
+
+h3(#app). Setup an identity provider's app
+
+To "group permissions":#permissions and "provision":#provision users with SCIM you need to enable "SSO":/account/sso with your chosen identity provider.
+
+h3(#group). Group permissions via SCIM
+
+When you enable "SSO":/account/sso with your chosen identity provider, you assign users and groups to the new identity provider's app. The group you created in identity provider's app is available to view and for user access management within Ably:
+
+* Open the *Account* navigation dropdown.
+** Click *Organization Users*.
+** Under *Ably Realtime identity provider groups*, click *Manage account access*.
+** Select the user's account access you want to manage.
+** Specify the user's required *Permissions* for the organization.
+** Assign the necessary *Roles* for required *Account* access.
+
+
+<aside data-type='note'>
+<p>When you adjust a user's individual roles and permissions, permissions assigned via groups appear greyed out, indicating they cannot be modified individually. </p>
+</aside>
+
+h3(#provision). Provision users
+
+Provisioning users enables you to grant multiple users access to your organization, either automatically via "SCIM":#scim or "manually":#manual through user invitations.
+
+<aside data-type='note'>
+<p>Provisioning users via SSO is recommended over manual invitation because it strengthens security by centralizing access control, enhances operational efficiency through automation, and improves the user experience with quicker access. Provisioning users via SSO also ensures compliance with regulatory requirements, and reduces the risk of human error.</p>
+</aside>
```

Comment:
```suggestion
Provisioning users via SCIM is recommended over manual invitation. This is because it centralizes access control to improve security, and offers automation to increase efficiency and reduce the risk of human error.
```